catboost/app/modes.h

```diff
@@ -7,3 +7,4 @@ int mode_metadata(int argc, const char* argv[]);
 int mode_run_worker(int argc, const char* argv[]);
 int mode_roc(int argc, const char* argv[]);
 int mode_model_sum(int argc, const char* argv[]);
+int mode_snapshot_to_model(int argc, const char* argv[]);
```

Comment:
The most important thing is to have it in python. So I would suggest adding this to catboost.load_model(format='CpuSnapshot')